### DISCUSSION TITLE
Facing a Problem with the Connector Build

### DISCUSSION THREAD

**Discussion by OmarAlKarghood** [eclipse-edc/Connector#3124] [Q&A]
Hi all,

I am new to EDC and was trying to build the connector on my Linux Subsystem on Win 11, and although the build is successful I do not see the jar file that I am supposed to run afterwards per the instructions on the tutrioal video made by Paul (https://youtu.be/O-YmJJp0nk8), I am missing the "basic" folder under "launchers", did the structure of the build change or am I missing something?(maybe the build was not successful? since I saw a lot of warnings)

Also, if someone can post their "build.gradle.kts" file to be sure that my setup is correct since I have been facing issues with the word "implementation" being unresolvable.

*build was done on Win 11 WSL using JDK 17 and Gradle 8.1.1 using the latest version of the code from GIT
**I have attached the build log in case it helps.
[logg.txt](https://github.com/eclipse-edc/Connector/files/11607415/logg.txt)

**Comment by paullatzelsperger** [comment]
Hi, unfortunately tech youtube videos in general - and those about incubating code in particular - don't age well. 
In the [Eclipse Dataspaces Components](https://github.com/eclipse-edc) org here you should find many examples of projects using the Connector at artefact level, in particular take a look at the Samples and the MinimumViableDataspace.

The logfile you attached indicates a successful build.

**Comment by tnolden** [comment]
Have you checked the recommended build.gradle.kts ?

https://github.com/eclipse-edc/Connector/blob/main/docs/developer/build-your-own-connector.md
